### PR TITLE
Content changes and formatting part 2 of HOTT-4070

### PR DESCRIPTION
**Pull request by shjohnson**
### Jira link

HOTT-4070

### What?

I have added/removed/altered:

- [ ] Remove extra content in the header
- [ ] Move the extra note under the bullet points and add tests

Screenshot 2023-10-13 at 11 31 11

### CODE REVIEW

File: app/views/steps/trader_scheme/show.html.erb [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
   <%= f.govuk_collection_radio_buttons :trader_scheme, @step.options, :id, :name, legend: { text: "#{trader_scheme_header}?", size: 'xl', tag: 'h1' }, hint: { text: "#{trader_scheme_body}" }, include_hidden: true %>
 
+  <p><%= trader_scheme_body_extra_text %></p>
```
Comment (alexdesi) [inline app/views/steps/trader_scheme/show.html.erb:10]: This way, we might have an empty paragraph. Should we put <p> paragraph inside the  trader_scheme_body_extra_text method.

File: app/helpers/ukims_helper.rb [hunk 2]
```diff
@@ -22,8 +22,15 @@ def trader_scheme_body
...
     else
-      "If you were moving goods into Northern Ireland which are for sale to, or final use by, end consumers located in the UK and you were authorised under the UK Trader Scheme or the UK Internal Market Scheme, then you may declare your goods as being 'not at risk' where the requirements are met. A 'not at risk' good entering Northern Ireland from Great Britain will not be subject to duty.
-       Please note that UK Internal Market scheme trades cannot benefit from expanded processing rules before 30 September 2023. Trades before this date will use the UK Trader Scheme rules on processing"
+      "If you were moving goods into Northern Ireland which are for sale to, or final use by, end consumers located in the UK and you were authorised under the UK Trader Scheme or the UK Internal Market Scheme, then you may declare your goods as being 'not at risk' where the requirements are met. A 'not at risk' good entering Northern Ireland from Great Britain will not be subject to duty."
+    end
+  end
+
+  def trader_scheme_body_extra_text
```
Comment (alexdesi) [inline app/helpers/ukims_helper.rb:29]: what about renaming it to something more evident?

maybe ` internal_market_scheme_note`

**Review by alexdesi — APPROVED** [review]
It looks good, just a couple of comments.

File: app/views/steps/trader_scheme/show.html.erb (hunk 1 above)
Comment (shjohnson) [reply app/views/steps/trader_scheme/show.html.erb:10]: Yeah true i did think about this

**Review by willfish — DISMISSED** [review]

**Review by willfish — APPROVED** [review]

**Pull request merged**